terraform/modules/gro_ingestion_service/lambdas/src/functions/deleteEvent.function.ts

```diff
@@ -0,0 +1,54 @@
+import { EventRequest } from "../models/EventRequest"
+import { DeleteEventResponse } from "../models/DeleteEventResponse"
+import { DynamoDBClient, DeleteItemCommand, DeleteItemInput } from "@aws-sdk/client-dynamodb"
+import { unmarshall } from "@aws-sdk/util-dynamodb"
+
+const tableName = process.env.TABLE_NAME ?? ""
+
+const dynamo = new DynamoDBClient({ apiVersion: "2012-08-10" })
+
+export const handler = async (event: EventRequest): Promise<DeleteEventResponse> => {
+
+    const params: DeleteItemInput = {
+        Key: {
+            hash: {
+                S: event.id,
+            }
+        },
+        TableName: tableName,
+        ReturnValues: "ALL_OLD",
+    }
+    try {
+        const command = new DeleteItemCommand(params)
+        const result = await dynamo.send(command)
+
+        let logParams = {
+            hash: event.id,
+            RegistrationId: "",
+            EventTime: "",
+        }
+        if (result.Attributes) {
```

Comment:
How would we get to the case where there are no Attributes here?

Reply:
It was just a edge case where the attributes where not returned but probs not possible